.github/workflows/publish.yaml

```
@@ -0,0 +1,94 @@
+# Build and publish to PyPI
```

```suggestion
name: Build and publish to PyPI
```

Should name the workflow a as a whole, or else it displays weird in the PR checks